vulnerable_people_form/integrations/postcode_eligibility.py

```diff
@@ -1,6 +1,15 @@
 import os
 
+from .persistence import get_rds_data_client, generate_string_parameter, execute_sql
+
 
 def check_postcode(postcode):
-    # TODO: add implementation
-    return os.environ.get("FAIL_POSTCODE_VERIFICATION") is None
+    records = execute_sql(
+        "CALL cv_base.is_postcode_on_lockdown(:postcode)",
```

Comment:
The version that looks like it is going through a PR/draft is "CALL cv_base.is_postcode_in_lockdown(:postcode)" so i am guessing this will change to that ?

Reply:
Whoops, that got reverted in a rebase. Good spot! Fixed.